Description: Initial fix for issue with not finding old segment files.
 - Increase maximum nodelist segment age to 7 weeks in src/makenl.c
 - Also, bump version to 3.3.4 in src/version.h.
Author: Andrew Leary <ajleary@users.sourceforge.net>
Bug: http://sourceforge.net/p/makenl/bugs/9
Origin, upstream: http://makenl.cvs.sourceforge.net/viewvc/makenl/makenl/src/makenl.c?revision=1.15
Origin, vendor: http://sourceforge.net/p/makenl/code, 746334092b30c99ad4b16e79e492e1a959750f74
Last-Update: 2013-02-06

---
 src/makenl.c  |    4 ++--
 src/version.h |    4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/makenl.c b/src/makenl.c
index d59c06d..b4cdf44 100644
--- a/src/makenl.c
+++ b/src/makenl.c
@@ -1,4 +1,4 @@
-/* $Id: makenl.c,v 1.14 2012/10/14 15:24:21 ozzmosis Exp $ */
+/* $Id: makenl.c,v 1.15 2013/01/30 21:58:10 ajleary Exp $ */
 
 #include <stdio.h>
 #include <time.h>
@@ -189,7 +189,7 @@ int main(int argc, char *argv[])
     mklog(LOG_INFO, "Cmdline: %s", cmdline_to_str(argv));
     mklog(LOG_INFO, "Using '%s' in '%s'", CfgFile, CurDir);
 
-    for (OldWeeks = 3; OldWeeks >= 0; OldWeeks--)
+    for (OldWeeks = 7; OldWeeks >= 0; OldWeeks--)
     {
         searchdow(NewExtWDay, -7 * OldWeeks + 6, &SplitTimePtr);
         sprintf(OldExtensions[OldWeeks], "%03d",
diff --git a/src/version.h b/src/version.h
index d7ace9d..92e3939 100644
--- a/src/version.h
+++ b/src/version.h
@@ -1,8 +1,8 @@
-/* $Id: version.h,v 1.18 2013/01/17 14:41:26 jame Exp $  */
+/* $Id: version.h,v 1.19 2013/01/30 21:58:10 ajleary Exp $  */
 
 #include "os.h"
 
-#define MAKENL_VERSION      "3.3.3"
+#define MAKENL_VERSION      "3.3.4"
 
 #define MAKENL_LONG_VERSION "MakeNL " MAKENL_VERSION " (" MAKENL_OS ") compiled with " MAKENL_CC " on " __DATE__ " " __TIME__
 
-- 
1.7.10.4

